Userland/Libraries/LibCodeComprehension/Cpp/Tests/complete_includes.cpp

```diff
@@ -1,5 +1,4 @@
 #include "sample_heade
 #include <sys/cdef
 
-void foo() {}
-
+void foo() { }
```

Comment:
hm I thought cpp tests would be omitted from clang-format, is that just the ones in Tests/LibCPP? also, why is this test here and not in Tests/ 🤔

Reply:
Good catch, added a commit similar to https://github.com/SerenityOS/serenity/commit/42865b8975818116b922aa2411eb167618fc532c and reverted this change

No idea about the second question :)